### PR TITLE
2731: Add linting rule for arrays

### PR DESCRIPTION
**Pull request by simomps**
### Short description

Add a linting style for arrays

### Proposed changes

- All arrays displayed as arrayName[]
- other array style changed according to new rule

### Side effects

- None

### Testing

Check that all settings still work.

### Resolved issues

Fixes: [#2731](https://github.com/orgs/digitalfabrik/projects/2/views/3?pane=issue&itemId=56983718&issue=digitalfabrik%7Cintegreat-app%7C2731) 

---

### CODE REVIEW

File: tools/yarn.lock [hunk 1]
```diff
@@ -136,40 +136,37 @@
     universal-github-app-jwt "^2.2.0"
     universal-user-agent "^7.0.0"
 
-"@octokit/auth-oauth-app@^7.1.0":
```
Comment (steffenkleinle) [inline tools/yarn.lock:139]: Do you know why these changes of the `yarn.lock` are present here in your PR? Usually, this file should only be modified if dependencies in a package.json are added/removed/upgraded.
Please revert this file if you did not actively or purposefully changed this file.

**Review by steffenkleinle — COMMENTED** [review]
Congrats to your first PR, nicely done :rocket: Just one small comment regarding the `yarn.lock`, I think this is otherwise good to go!

File: tools/yarn.lock (hunk 1 above)
Comment (simomps) [reply tools/yarn.lock:139]: Ah no. I didn't change it on purpose. I will revert the changes. 
Thanks for the notice :)

**Review by steffenkleinle — APPROVED** [review]
Thank you :+1:

**Review by LeandraH — COMMENTED** [review]
Very cool, thank you very much for cleaning up  🧽 

I have found two more places where we still use the `Array` type, one is `Selector.ts` in the e2e-tests, the other one in `helpers.spec.ts` in native, might be worth a check why the linter doesn't find that. But then again, maybe not 😂

**Review by LeandraH — APPROVED** [review]
I'll approve it already since I'll be on vacation for a bit and don't want to block anything :)

**Comment by steffenkleinle** [comment]
> Very cool, thank you very much for cleaning up 🧽
> 
> I have found two more places where we still use the `Array` type, one is `Selector.ts` in the e2e-tests, the other one in `helpers.spec.ts` in native, might be worth a check why the linter doesn't find that. But then again, maybe not 😂

Actually I am quite sure that this is something different. The two occurrences mentioned by you are both referring to
something like the following:
``` ts
const keys = new Array<string>()
```

These are initialization of a new array instance (of type string), opposed to something like the following  which actually throws an error in both cases:
``` ts
const keys: Array<string> = new Array<string>()
```

I therefore think we are actually good to go here :)

**Comment by LeandraH** [comment]
Woooops, you're right, sorry!
